src/vos/evtree.c

```diff
@@ -1288,15 +1413,15 @@ evt_node_insert(struct evt_context *tcx, struct evt_node *nd, umem_off_t in_off,
 	int		 rc;
 	bool		 changed = 0;
 
-	V_TRACE(DB_TRACE, "Insert "DF_RECT" into "DF_RECT"\n",
-		DP_RECT(&ent->ei_rect), DP_RECT(evt_node_mbr_get(tcx, nd)));
+	V_TRACE(DB_TRACE, "Insert "DF_RECT" into "DF_RECT_DF"\n",
+		DP_RECT(&ent->ei_rect), DP_RECT_DF(evt_node_mbr_get(nd)));
 
 	rc = tcx->tc_ops->po_insert(tcx, nd, in_off, ent, &changed, csum_bufp);
 	if (rc != 0)
 		return rc;
 
-	V_TRACE(DB_TRACE, "New MBR is "DF_RECT", nr=%d\n",
-		DP_RECT(evt_node_mbr_get(tcx, nd)), nd->tn_nr);
+	V_TRACE(DB_TRACE, "New MBR is "DF_RECT_DF", nr=%d\n",
+		DP_RECT_DF(evt_node_mbr_get(nd)), nd->tn_nr);
```

Comment:
(style)  'nd' may be misspelled - perhaps 'and'?
(style)  'nd' may be misspelled - perhaps 'and'?